returnn/oggzip.py

```diff
@@ -78,20 +81,26 @@ def run(self):
         if self.segments is not None:
             args.extend(["--subset_segment_file", tk.uncached_path(self.segments)])
 
-        if self.no_conversion:
-            args.extend(["--no_conversion"])
+        if self.no_audio:
+            args.extend(["--no_ogg"])
         else:
-            if self.rasr_cache is not None:
-                args.extend(["--sprint_cache", tk.uncached_path(self.rasr_cache)])
-            if self.raw_sample_rate is not None:
-                args.extend(["--raw_sample_rate", str(self.raw_sample_rate)])
-            if self.feat_sample_rate is not None:
-                args.extend(["--feat_sample_rate", str(self.feat_sample_rate)])
+            if self.no_conversion:
+                args.extend(["--no_conversion"])
+            else:
+                if self.rasr_cache is not None:
+                    args.extend(["--sprint_cache", tk.uncached_path(self.rasr_cache)])
+                if self.raw_sample_rate is not None:
+                    args.extend(["--raw_sample_rate", str(self.raw_sample_rate)])
+                if self.feat_sample_rate is not None:
+                    args.extend(["--feat_sample_rate", str(self.feat_sample_rate)])
 
         sp.check_call(args)
         relink("out.ogg.zip", self.out_ogg_zip.get_path())
 
     @classmethod
     def hash(cls, parsed_args):
         del parsed_args["returnn_python_exe"]
+        # hash compatibility
+        if parsed_args["no_audio"] == False:
+            del parsed_args["no_audio"]
```

Comment:
Would you not rather do this via a `__sis_hash_exclude__` instead of coding in the hash function such that it can be found easier for future hash correction dates and/or is more obvious why the `no_audio == False` case is treated specially.

Reply:
It seems I combined the wrong commits into the branch :D This does not belong here...